jerry-core/jmem/jmem-allocator.c

```diff
@@ -188,6 +188,10 @@ jmem_compress_pointer (const void *pointer_p) /**< pointer to compress */
 #if defined (ECMA_VALUE_CAN_STORE_UINTPTR_VALUE_DIRECTLY) && defined (JERRY_CPOINTER_32_BIT)
   JERRY_ASSERT (((jmem_cpointer_t) uint_ptr) == uint_ptr);
 #else /* !ECMA_VALUE_CAN_STORE_UINTPTR_VALUE_DIRECTLY || !JERRY_CPOINTER_32_BIT */
+#ifdef JERRY_ENABLE_EXTERNAL_CONTEXT
+  JERRY_DEFINE_CURRENT_CONTEXT ();
+  JERRY_UNUSED (jerry_current_external_context);
+#endif
```

Comment:
What's the point of this? If this approach needs to be guarded by feature macros at the macro expansion sites then this is going to be a maintenance nightmare. But here I don't even see what this is trying to accomplish.